wp-content/plugins/zippicks-favorites/includes/class-analytics.php

```
@@ -0,0 +1,344 @@
+<?php
+namespace ZipPicks\Favorites;
+
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+class Analytics {
+    
+    public function get_analytics_data() {
+        global $wpdb;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$wpdb'. (Unused Code Rules)

(UnusedLocalVariable)